utils/srcmanip/check_import_usage

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+"""Simple script to ckeck usage of imports in Fortran files"""
+
+import argparse
+import fnmatch
+import os
+import re
+
+
+_DESCRIPTION = "Checks the imports in DFTB+ Fortran file(s) for usage"
+
+_PAT_USE_MODULE = re.compile(
+    r"""^(?P<indent>[ ]*)use
+    (?P<attrib>(?:\s*,\s*intrinsic)?)
+    (?P<separator>[ ]*::[ ]*|[ ]*)
+    (?P<name>\w+)
+    (?P<rest>.*?(?:&[ ]*\n(?:[ ]*&)?.*?)*)\n
+    """,
+    re.VERBOSE | re.MULTILINE,
```

Comment:
Maybe we should add also `re.IGNORECASE` (as we're dealing with Fortran :wink: )